src/components/Pagination/Pagination.tsx

```diff
@@ -7,6 +7,20 @@ import Select from '../Select/Select';
 import ChevronRight from '../../assets/chevronRight.svg?react';
 import IconButton from '../IconButton/IconButton';
 import { ButtonVariants } from '../Button/types';
+import { determineOptionValue, findOptionFromSelectString } from './utils';
+
+export type PaginationOptionValue = string | number;
+
+export interface PaginationOption {
```

Comment:
dup of changes in https://github.com/PhillipsAuctionHouse/seldon/pull/340#pullrequestreview-2343217552